frontend/src/components/datasets/DatasetsListContent.tsx

```diff
@@ -30,22 +31,23 @@ function DatasetsListContent() {
 
 	const callbacks = usePagedItemsActionsCallbacks(DatasetsTableActions)
 
-	const columns = useFilteredColumns(
-						tableColumns, 
-						DATASET_FILTER_DEFINITIONS,
-						DATASET_FILTER_KEYS,
-						filters,
-						callbacks.setFilterCallback,
-						callbacks.setFilterOptionsCallback)
+  let tweakedColumns = useFilteredColumns(
+          tableColumns,
```

Comment:
why did you rename from columns to tweakedColumns?

Reply:
i tried mirroring another component file as reference. it didn't quite work and i was able to reach a simpler way

Reply:
I think Nafiz mention this because the change of variable name to "tweakedColumns" do not really carry more information than the simpler "columns" that was already there.